src/Features/Core/Portable/MakeMethodSynchronous/AbstractMakeMethodSynchronousCodeFixProvider.cs

```diff
@@ -81,20 +86,167 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
             return newSolution;
         }
 
-        private async Task<Solution> RemoveAsyncTokenAsync(Document document, IMethodSymbol methodSymbolOpt, SyntaxNode node, CancellationToken cancellationToken)
+        private async Task<Solution> RemoveAsyncTokenAsync(
+            Document document, IMethodSymbol methodSymbolOpt, SyntaxNode node, CancellationToken cancellationToken)
         {
             var compilation = await document.Project.GetCompilationAsync(cancellationToken).ConfigureAwait(false);
             var taskType = compilation.GetTypeByMetadataName("System.Threading.Tasks.Task");
             var taskOfTType = compilation.GetTypeByMetadataName("System.Threading.Tasks.Task`1");
 
+            var annotation = new SyntaxAnnotation();
             var newNode = RemoveAsyncTokenAndFixReturnType(methodSymbolOpt, node, taskType, taskOfTType)
-                .WithAdditionalAnnotations(Formatter.Annotation);
+                .WithAdditionalAnnotations(Formatter.Annotation, annotation);
 
             var root = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
             var newRoot = root.ReplaceNode(node, newNode);
 
             var newDocument = document.WithSyntaxRoot(newRoot);
-            return newDocument.Project.Solution;
+            var newSolution = newDocument.Project.Solution;
+
+            if (methodSymbolOpt == null)
+            {
+                return newSolution;
+            }
+
+            return await TryRemoveAwaitFromCallersAsync(
+                newDocument, annotation, cancellationToken).ConfigureAwait(false) ;
+        }
+
+        private async Task<Solution> TryRemoveAwaitFromCallersAsync(
+            Document document, SyntaxAnnotation annotation, CancellationToken cancellationToken)
+        {
+            var syntaxRoot = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+            var methodDeclaration = syntaxRoot.GetAnnotatedNodes(annotation).FirstOrDefault();
+            if (methodDeclaration != null)
+            {
+                var semanticModel = await document.GetSemanticModelAsync(cancellationToken).ConfigureAwait(false);
+                var methodSymbol = semanticModel.GetDeclaredSymbol(methodDeclaration) as IMethodSymbol;
+
+                if (methodSymbol != null)
+                {
+                    var references = await SymbolFinder.FindRenamableReferencesAsync(
```

Comment:
I thought about that, but it's much harder to use the original results.  We would need to go and track all the rename locations through the rewrite.  This approach seemed much simpler and safer.